### PR TITLE
Day & Tower Input Parameters.

### PR DESCRIPTION
**Pull request by udai-arneja**
Ticks per day now dependent on other variables.

# Summary

This PR is aiming to solve the issue #13 . Day and tower parameters as set from discussion in meeting. 

## Additional Information

From meeting: input params: composition of agents (and therefore total number of agents), number of agents per floor (this will need to be a factor of the total number of agents), time spent per floor. (Courtesy of @jzzheng22 )

## Test Plan

Tested by observing logs of reshuffling etc while changing `ticksPerFloor` in main.go - changes day length as expected.

### CODE REVIEW

**Review by Rhys-J — COMMENTED** [review]
Thanks Udai!
LGTM, probably best to wait for other people to review before merging to main in case they want changes.

File: cmd/backend/main.go [hunk 1]
```diff
@@ -60,16 +60,18 @@ func main() {
...
-	simDays := 3
-	reshuffleDays := 1
-	dayInfo := day.NewDayInfo(ticksPerDay, simDays, reshuffleDays)
-
 	// can have frontend parameters come go straight into simEnv
 	foodOnPlatform := 100.0
 	numOfAgents := []int{2, 2, 2}
 	agentHP := 100
 	agentsPerFloor := 1 //more than one not currently supported
+	numberOfFloors := simulation.Sum(numOfAgents) / agentsPerFloor
```
Comment (JaafarRammal) [inline cmd/backend/main.go:68]: I wasn't at the meeting so I can't comment on the decision of which variables are inputs, but what's the guarantee `numberOfFloors` is an integer here?

**Review by JaafarRammal — APPROVED** [review]

**Review by jzzheng22 — DISMISSED** [review]
LGTM, thanks so much

**Comment by udai-arneja** [comment]
Sorry for the re-reviews, needed to resolve a conflict and I think it automatically makes the reviews stale?

**Review by jzzheng22 — APPROVED** [review]

**Pull request merged**